packages/bpk-component-nudger/src/themeAttributes.js

```diff
@@ -16,6 +16,8 @@
  * limitations under the License.
  */
 
+/* @flow strict */
```

Comment:
We don't normally flow type these themeAttributes file, is there a reason for doing so?

Reply:
Actually ignore me!